pkg/rtc/uptrackmanager.go

```diff
@@ -415,7 +422,9 @@ func (u *UpTrackManager) maybeAddPendingSubscription(trackID livekit.TrackID, su
 	}
 
 	u.pendingSubscriptions[trackID] = append(u.pendingSubscriptions[trackID], subscriberIdentity)
-	go sub.SubscriptionPermissionUpdate(u.params.SID, trackID, false)
+	u.opsQueue.Enqueue(func() {
+		sub.SubscriptionPermissionUpdate(u.params.SID, trackID, false)
```

Comment:
subscription permission updates were getting out of order. Was not the reason for the quick switching permissions issue, but would have been confusing to clients.

Reply:
nice!